### ISSUE TITLE
BUG - `Cannot read properties of undefined (reading 'includes')` error thrown when trying access to `UI` annotation

### ISSUE THREAD

**Issue by 815are** [SAP/open-ux-odata#862]
### Steps to Reproduce
1. Project -> <Redacted>
2. There is following [annotation](<Redacted>
```
 <Annotations Target="LibraryService.LibraryElement">
        <Annotation Term="UI.CreateHidden">
          <Path>/AuthorizationRestriction/isNotAllowedToCreate</Path>
        </Annotation>
...
```
3. when we try to access it like using code like:
```
oDataServiceAVT.resolvePath('/LibraryElement').target.annotations.UI.CreateHidden
```
4. Error is thrown from @sap-ux/annotation-converter ->https://github.com/SAP/open-ux-odata/blob/main/packages/annotation-converter/src/converter.ts#L422

From debugger:
![image](https://github.com/user-attachments/assets/e9312071-6140-4787-9a47-3b5aaad25c29)

### Expected results
Error is not thrown

### Actual results
Error is thrown

### Screenshots
![image](https://github.com/user-attachments/assets/b4aabbba-3733-4592-b5cb-de69ab38fd53)

![image](https://github.com/user-attachments/assets/e9312071-6140-4787-9a47-3b5aaad25c29)

#### Version/Components/Environment	
Add any other context about the problem here
OS:                      
- [x] Mac OS
- [x] Windows
- [ ] Other

### Problem
{describe the problem} 

### Fix
{describe the fix} 

### Why was it missed
{Some explanation why this issue might have been missed during normal development/testing cycle}  

### How can we avoid this
{if we don’t want to see this type of issues anymore what we should do to prevent}

**Comment by nlunets** [comment]
Fixed with  [@sap-ux/edmx-parser@0.8.1](https://github.com/SAP/open-ux-odata/releases/tag/%40sap-ux%2Fedmx-parser%400.8.1)

**Comment by schreckstefan** [comment]
Thanks, @nlunets. 
But pls. note that I had to adapt our copy of the StableIdHelper for this, evaluation of odata.concat
